lib/ae_mdw/util.ex

```diff
@@ -169,4 +169,17 @@ defmodule AeMdw.Util do
         map
     end
   end
+
+  @spec convert_params(
+          map(),
+          ({binary(), binary()} -> {:ok, {atom(), term()}} | {:error, term()})
+        ) :: {:ok, Keyword.t()} | {:error, term()}
```

Comment:
5 cents on a reusable `convert_params`: instead of returning a `Keyword.t()` it could return a `map()` since each context module instantiates it.

Reply:
Done